golem-common/src/model/plugin.rs

```diff
@@ -271,6 +316,115 @@ pub trait PluginScope:
     async fn accessible_scopes(&self, context: Self::RequestContext) -> Result<Vec<Self>, String>;
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(feature = "poem", derive(poem_openapi::Object))]
+#[cfg_attr(feature = "poem", oai(rename_all = "camelCase"))]
+#[serde(rename_all = "camelCase")]
+pub struct ProjectPluginScope {
+    pub project_id: ProjectId,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "type")]
+#[cfg_attr(feature = "poem", derive(poem_openapi::Union))]
+#[cfg_attr(feature = "poem", oai(discriminator_name = "type", one_of = true))]
+pub enum CloudPluginScope {
+    Global(Empty),
```

Comment:
Is `Global(Empty)` <=> `Global`?

Reply:
This is because of poem-openapi, you need to have a "body" otherwise you cannot have a `Union`

Reply:
:100: